src/components/ValueCard/ValueCard.story.jsx

```diff
@@ -160,6 +160,31 @@ storiesOf('Watson IoT/ValueCard', module)
       </div>
     );
   })
+  .add('small / long string', () => {
+    const size = select('size', Object.keys(CARD_SIZES), CARD_SIZES.SMALL);
+    return (
+      <div style={{ width: text('cardWidth', `${getCardMinSize('lg', size).x}px`), margin: 20 }}>
+        <ValueCard
+          title="Tagpath"
+          id="facilitycard"
+          content={{
+            attributes: [
+              {
+                label: 'Tagpath',
+                dataSourceId: 'footTraffic',
+              },
+            ],
+          }}
+          breakpoint="lg"
+          size={size}
+          values={{
```

Comment:
Could we make this a knob so users could try can more immediately see the difference with a shorter string?
There's an example in the above stories

Reply:
Added.